docs/general/status.rst

```diff
@@ -0,0 +1,89 @@
+SecureDrop Workstation Project Status
+=====================================
+
+SecureDrop Workstation is currently in active development, and has not yet
+had a stable release.
+
+The 1.0.0 general availability release is tentatively planned for TBD.
```

Comment:
Is the `TBD` placeholder to be updated (a) before this is published or (b) at a later date?

Reply:
The intention is for (a), I just wanted to be careful and not publicly commit to time frame that we don't feel comfortable about.